sdk/ml/azure-ai-ml/azure/ai/ml/_schema/job/queue_settings.py

```diff
@@ -0,0 +1,27 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+from marshmallow import fields, post_load
+from azure.ai.ml.constants._job.job import JobPriorityValues, JobTierNames
+from azure.ai.ml._schema.core.fields import StringTransformedEnum
+from ..core.schema import PathAwareSchema
+
+from .resource_configuration import ResourceConfigurationSchema
+
+
+class QueueSettingsSchema(PathAwareSchema):
```

Comment:
I don't see this schema being added to command/sweep job schema. Is that expected?

Reply:
The changes will be part of Vijetha's PR or I will send a follow up PR.